libs/sdk-ui-kit/src/FlexDimensions/FlexDimensions.tsx

```diff
@@ -0,0 +1,78 @@
+// (C) 2007-2020 GoodData Corporation
+import React, { Component, createRef, ReactNode, RefObject } from "react";
+import cx from "classnames";
+import throttle from "lodash/throttle";
+import pickBy from "lodash/pickBy";
+
+import { elementRegion } from "../utils/domUtilities";
+
+import { IFlexDimensionsProps, IFlexDimensionsState } from "./typings";
+
+/**
+ * @internal
+ */
+export class FlexDimensions extends Component<IFlexDimensionsProps, IFlexDimensionsState> {
```

Comment:
Is this component even necessary? What does it add compared to react-measure we use elsewhere?

Reply:
It seems to work bit differently. I will create task for this and get rid of it in the next round of the refactors. It is used in legacy lists and in KD.

Reply:
👍  thanks